docs/php.md

```diff
@@ -14,6 +14,9 @@ xdebug.start_with_request=yes
 xdebug.log_level = 0
 ```
 
+This can be useful for also debugging php invoked from the CLI.
```

Comment:
With "useful" you mean "This also works for debugging php invoked from the CLI"?

If you have the same PHP.ini for web and cli then indeed it will be applied to both and also enabled for both. Unless you have a different configuration for php on CLI.